ticdc/ticdc-debezium.md

```diff
@@ -0,0 +1,138 @@
+---
+title: TiCDC Debezium Protocol
+summary: Learn the concept of the TiCDC Debezium Protocol and how to use it.
+---
+
+# TiCDC Debezium Protocol
+
+[Debezium](https://debezium.io/) is a tool for capturing database changes. It converts each captured database change into a message called an "event" and sends these events to Kafka. Starting from v8.0.0, TiCDC supports sending TiDB changes to Kafka using a Debezium style output format, simplifying migration from MySQL databases for users who had previously been using Debezium's MySQL integration.
+
+## Use the Debezium message format
+
+When you use Kafka as the downstream sink, specify the `protocol` field as `debezium` in `sink-uri` configuration. Then TiCDC encapsulates the Debezium message based on the event and send TiDB data change events to the downstream.
+
+Currently, the Debezium protocol only supports Row changed events and directly ignores DDL events and WATERMARK events. A Row changed event represents a change in a row of data. When a row changes, the Row changed event is sent, including relevant information about the row both before and after the change.
```

Comment:
```suggestion
Currently, the Debezium protocol only supports Row changed events and directly ignores DDL events and WATERMARK events. A Row changed event represents a data change in a row. When a row changes, the Row changed event is sent, including relevant information about the row both before and after the change. A WATERMARK event marks the replication progress of a table, indicating that all events less than the watermark have been sent to the downstream.
```

Reply:
Do we need to use "Row Changed Event" as [TiCDC Open Protocol | PingCAP Docs](https://docs.pingcap.com/tidb/dev/ticdc-open-protocol) does?

![image](https://github.com/pingcap/docs/assets/79440533/3651f474-214b-4ace-9ac2-e3749bb26527)

Reply:
Although I don't think it's necessary to capitalize all words, we can keep its usage consistent among different docs pages.